packages/stream_video/lib/src/webrtc/rtc_manager.dart

```diff
@@ -714,6 +714,21 @@ extension RtcManagerTrackHelper on RtcManager {
 
     // Track found, mute/unmute it.
     if (track != null) {
+      if (enabled &&
+          track.trackType == SfuTrackType.screenShare &&
```

Comment:
Can you somehow encapsulate this long condition behind a getter?

Reply:
Done